content/post/66-london-hack-week-report.md

```diff
@@ -0,0 +1,56 @@
+---
+date: 2019-01-24
+url: 65-london-hack-week-report
+tags: hack week
+title: 2018 Q4 London Hack Week Summary
+author: David Dias
+---
+
+Back in October last year, the Go Core Dev Team for the IPFS, IPLD and libp2p projects spent some quality time together in London for a Hack Week. The overall feeling was excitment as it was the first time that many of the core contributors got the chance to meet in person. 
+
+Together, we explored our goals for the projects, along with the users needs and who is pushing for them. It was a great gathering and we wanted to make sure to share the results and the recordings with you.
+
+The week format was really interesting and one that could almost go into the Async Team Management books (as a way to create sync time for distributed teams). A simplified overview of what happened is:
+
+- **Monday** - Collaboratively surface the top priorities for the project. We defined them clearly and worked with the IPFS Project Working Group to prioritize them with the IPFS roadmap in mind.
+- **Tuesday** - A ton of hacking and design discussions. Let the Hackers be free!
+- **Wednesday** - Objectived and Key Restults (OKR) check in. We went from a "top down" (few to many) OKR approach to a "divide and conquer". The 3 main areas of focus were identified (Data Exchange, Datastore & Providers) and the big go-ipfs team was divided into 3 groups that worked independently for 60 minutes to review and came up with great improvements to the original OKRs that had been defined before we met.
```

Comment:
"Objectived and Key Restults" => "Objectives and Key Results"

Reply:
good catch. fixed! :)